src/PollbotAPI.js

```diff
@@ -6,24 +6,26 @@ import type {
   ReleaseInfo,
 } from './types.js';
 
-export function getOngoingVersions(): Promise<OngoingVersions> {
-  return fetch(
+export async function getOngoingVersions(): Promise<OngoingVersions> {
+  const response = await fetch(
     'https://pollbot.dev.mozaws.net/v1/firefox/ongoing-versions',
-  ).then(resp => resp.json());
+  );
+  return response.json();
 }
 
-export function getReleaseInfo(version: string): Promise<ReleaseInfo> {
-  return fetch(
+export async function getReleaseInfo(version: string): Promise<ReleaseInfo> {
+  const response = await fetch(
     `https://pollbot.dev.mozaws.net/v1/firefox/${version}`,
```

Comment:
nit: it looks like the server part could be put in a constant ``${SERVER}/firefox/${version}``